app/controllers/rooms_controller.rb

```diff
@@ -24,17 +22,37 @@ def create
 
   def destroy
     room = Room.find(room_params[:id])
-    render json: room
-    room.users.destroy
-    room.user_rooms.destroy
-    room.room_questions.destroy
     room.destroy
```

Comment:
I believe you could set up these other deletes to all happen when you delete the room using the `dependent` option in the relationships.

https://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-dependent